src/status_im/communities/core.cljs

```diff
@@ -221,18 +220,11 @@
 (fx/defn create
   {:events [::create-confirmation-pressed]}
   [{:keys [db]}]
-  (let [{:keys [name description image]} (get db :communities/create)]
+  (let [{:keys [name description membership image]} (get db :communities/create)]
     ;; If access is ENS only, we set the access to require approval and set the rule
     ;; of ens only
-    (let [params {:name name
-                  :description description
-                  :membership constants/community-on-request-access
-                  :color (rand-nth colors/chat-colors)
-                  :image (string/replace-first (str image) #"file://" "")
-                  :imageAx 0
-                  :imageAy 0
-                  :imageBx crop-size
-                  :imageBy crop-size}]
+    (let [params ((assoc :membership constants/community-on-request-access
```

Comment:
Mmh, not sure, I think this should be the same as before, no changes are required, `ens-only` communities are not used at the moment, so you can keep the same as develop

Reply:
Roger that. Rolled back.